### ISSUE TITLE
how to disable integrations

### ISSUE THREAD

**Issue by LanceSandino** [echoboomer/incident-bot#176]
I thought by leaving them commented out or even as empty `{}` they would be disabled, but so far it seems that we must specify (even if not in use)

* jira
* confluence
* statuspage

**Comment by LanceSandino** [comment]
Interesting… it was complaining for confluence, jira and statuspage that no definitions were found. 

Is it possible tbe m code maybe looking at the env var being set or the config?

**Comment by echoboomer** [comment]
If you encounter this error after upgrading, let me know.

**Comment by LanceSandino** [comment]
I can't replicate this, so we can consider this closed.
Maybe it was a config issue or version issue.

**Comment by lholland421** [comment]
Hi Team, We have encountered this issue with a new install on version 1.4.2. 
We have not set any Atlassian keys so we were under the impression the integration shouldn't work. 
However on startup the service crashes with the below: 

```
INFO:__main__:Running startup tasks...
INFO:slack.client:Stored current Slack users in database...
Traceback (most recent call last):
  File "/incident-bot/main.py", line 147, in <module>
    startup_tasks()
  File "/incident-bot/main.py", line 81, in startup_tasks
    if "confluence" in config.active.integrations.get("atlassian"):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable
```

Rolling back to version 1.3.2 fixes this issue FYI.

**Comment by echoboomer** [comment]
Thanks for reporting that @lholland421 - would you mind opening a new issue reporting this bug? I'll take a look ASAP.

**Comment by echoboomer** [comment]
@lholland421 this has been fixed in version `v1.4.2` with chart version `1.4.2`. Thanks again for reporting it.
